server/scripts/standalone/publish-standalone.sh

```diff
@@ -10,8 +10,9 @@ build_artifact() {
     echo "[build]  Creating standalone extender server artifact at ${ARTIFACT_DIR}..."
     rm -rf ${ARTIFACT_DIR}
     mkdir -p ${ARTIFACT_DIR}
-    cp ${SERVER_DIR}/build/libs/extender-0.1.0.jar ${ARTIFACT_DIR}/extender.jar
-    cp ${SERVER_DIR}/manifestmergetool/build/libs/manifestmergetool-0.1.0.jar ${ARTIFACT_DIR}/manifestmergetool.jar
+    cp ${SERVER_DIR}/build/standalone/extender-0.1.0.jar ${ARTIFACT_DIR}/extender.jar
```

Comment:
These files are copied by the buildStandalone() gradle task to build/standalone so it's better to copy them from that location when we build the artifact